pkg/schedule/type/type.go

```diff
@@ -70,3 +70,46 @@ const (
 	// LabelScheduler is label scheduler name.
 	LabelScheduler CheckerSchedulerType = "label-scheduler"
 )
+
+// SchedulerTypeCompatibleMap temporarily exists for compatibility.
```

Comment:
I suggest adding more explanation about when we need to consider the compatibility.

Reply:
I don't think there is a need to explain in the code now. Its current role is only to do a layer of conversion when the user adds a scheduler. If it is used in more places in the future, or permanently exists in the code, I will add a more accurate description to it.

Reply:
ping @rleungx

Reply:
I just wondering when we need to consider the compatibility.

Reply:
This is an intermediate pr, it may be deleted in the subsequent pr, if it stays, I will add more comments